src/SectorView.cpp

```diff
@@ -402,119 +211,14 @@ void SectorView::SaveToJson(Json::Value &jsonObj)
-		return;
-
-	m_previousSearch = search;
-
-	//Try to detect if user entered a sector address, comma or space separated, strip parentheses
-	//system index is unreliable, so it is not supported
-	try {
-		GotoSector(SystemPath::Parse(search.c_str()));
-		return;
-	} catch (SystemPath::ParseFailure) {}
-
-	bool gotMatch = false, gotStartMatch = false, gotExactMatch = false;
-	SystemPath bestMatch;
-	std::vector<std::pair<SystemPath,std::string>> exactMatches;
-	const std::string *bestMatchName = 0;
-
-	for (auto i = m_sectorCache->Begin(); i != m_sectorCache->End(); ++i)
-	{
-		for (unsigned int systemIndex = 0; systemIndex < (*i).second->m_systems.size(); systemIndex++)
-		{
-			const Sector::System *ss = &((*i).second->m_systems[systemIndex]);
-
-			// compare with the start of the current system
-			if (strncasecmp(search.c_str(), ss->GetName().c_str(), search.size()) == 0)
-			{
-				// matched, see if they're the same size
-				if (search.size() == ss->GetName().size())
-				{
-					// exact match, take it and go
-					SystemPath path = (*i).first;
-					path.systemIndex = systemIndex;
-					exactMatches.push_back(std::make_pair(path,ss->GetName()));
-					gotExactMatch = true;
-					continue;
-				}
-
-				// partial match at start of name
-				if (!gotMatch || !gotStartMatch || bestMatchName->size() > ss->GetName().size())
-				{
-					// don't already have one or its shorter than the previous
-					// one, take it
-					bestMatch = (*i).first;
-					bestMatch.systemIndex = systemIndex;
-					bestMatchName = &(ss->GetName());
-					gotMatch = gotStartMatch = true;
-				}
-
-				continue;
-			}
-
-			// look for the search term somewhere within the current system
-			if (pi_strcasestr(ss->GetName().c_str(), search.c_str()))
-			{
-				// found it
-				if (!gotMatch || !gotStartMatch || bestMatchName->size() > ss->GetName().size())
-				{
-					// best we've found so far, take it
-					bestMatch = (*i).first;
-					bestMatch.systemIndex = systemIndex;
-					bestMatchName = &(ss->GetName());
-					gotMatch = true;
-				}
-			}
-		}
-	}
-
-	if(gotExactMatch)
-	{
-		// We have some exact matches, sort them by distance and choose the closest to the players current system
-		const SystemPath currentSector = m_current.SectorOnly();
-		double nearest = DBL_MAX;
-		for(auto eM : exactMatches)
-		{
-			const double dist = SystemPath::SectorDistanceSqr(currentSector, eM.first);
-			if(dist<nearest)
-			{
-				// this one's closer, store it's details
-				bestMatch = eM.first;
-				bestMatchName = &(eM.second);
-				gotMatch = true;
-			}
-		}
-	}
-
-	if (gotMatch)
-	{
-		m_statusLabel->SetText(stringf(gotExactMatch ? Lang::EXACT_MATCH_X : Lang::NOT_FOUND_BEST_MATCH_X, formatarg("system", *bestMatchName)));
-		GotoSystem(bestMatch);
-	}
-	else
-	{
-		m_statusLabel->SetText(Lang::NOT_FOUND);
-	}
-}
 
 #define FFRAC(_x)	((_x)-floor(_x))
 
 void SectorView::Draw3D()
 {
 	PROFILE_SCOPED()
 
-	m_lineVerts->Clear();
+		m_lineVerts->Clear();
```

Comment:
Ok last one I'll comment on but I think this is caused by the `PROFILE_SCOPED()` above which has no `;` at the end.